pan-domain-auth-core/src/main/scala/com/gu/pandomainauth/model/PanDomainAuthSettings.scala

```diff
@@ -33,8 +32,8 @@ object PanDomainAuthSettings{
 
   def apply(settingMap: Map[String, String]): PanDomainAuthSettings = {
     val cookieSettings = CookieSettings(
-      legacyCookieName = settingMap("legacyCookieName"),
-      assymCookieName = settingMap("assymCookieName")
+      // Try legacy setting name first for compatibility
+      cookieName = settingMap.getOrElse("assymCookieName", settingMap("cookieName"))
```

Comment:
I think this could be clearer if `assymCookieName` was a constant named `legacySettingName` or similar